src-ghc/Pact/PersistPactDb/Regression.hs

```diff
@@ -58,7 +59,7 @@ runRegression p = do
               ,("keyset" .= String "mod-admin-keyset")
               ,("name" .= String "mod1")
               ,("code" .= String "code")
-              ,("meta" .= object [("model" .= Null)
+              ,("meta" .= object [("model" .= Array (V.fromList []))
```

Comment:
No need to import vector just for this, use ([]::Text) just for it's toJSON instance here